docs/migration-guide/to-16.md

```diff
@@ -45,6 +45,15 @@ async function lint() {
 
 If you use `stylelint.lint()` to lint files, the `code` property will always be `undefined`.
 
+## Changed Node.js API returned `formatters` object
```

Comment:
minor

`…returned by the…`

Reply:
I followed the heading above:

```
## Changed Node.js API returned resolved object
```

https://github.com/stylelint/stylelint/pull/7184/files#diff-38349b678797293b4889d8dd738f084a2b19e1ba08b5de5cce31aa094f2b7a6eR26

But it seems they're not so good. Let me know if you have a better idea.